style-guide.md

```diff
@@ -0,0 +1,119 @@
+# Univerzális stílus útmutató
+
+## Kezdő lépések
+Kezdésnek olvasd el [ezt a kommentet](https://github.com/reactjs/hu.reactjs.org/issues/1#issue-445691844)
+
+## Fejléc azonosítók
+
+Minden fejlécnek van egy saját azonosítója:
+
+```md
+## Try React {#try-react}
+```
+
+Ezeket az azonosítókat **ne** fordítsd le! Ezek a navigáció szempontjából fontosak, és ha megváltozank, a külső linkek nem fognak működni, azaz:
+
+```md
+See the [beginning section](/getting-started#try-react) for more information.
+```
+
+✅ TEDD:
+
+```md
+## Próbáld ki a React-et {#try-react}
+```
+
+❌ NE TEDD:
+
+```md
+## Próbáld ki a React-et {#probald-ki-a-reactet}
+```
+
+Ez elérhetetlenné teszi a fenti linket.
+
+## Szöveg Kódblokkokban
+
+A kódblokkokban lévő szövegeket - a kommentek, esetleg a sztringek kivételével - ne fordítsd le. Vigyázz, hogy kódra utaló sztringeket ne fordíts le!
+
+Példa:
+```js
+// Example
+const element = <h1>Hello, world</h1>;
+ReactDOM.render(element, document.getElementById('root'));
+```
+
+✅ TEDD:
+
+```js
+// Példa
+const element = <h1>Hello, world</h1>;
+ReactDOM.render(element, document.getElementById('root'));
+```
+
+✅ EZ IS OK:
+
+```js
+// Példa
+const element = <h1>Helló világ</h1>;
+ReactDOM.render(element, document.getElementById('root'));
+```
+
+❌ NE TEDD:
+
+```js
+// Példa
+const element = <h1>Helló világ</h1>;
+// "root" refers to an element ID.
+// DO NOT TRANSLATE
+ReactDOM.render(element, document.getElementById('gyökér'));
+```
+
+❌ SEMMIKÉPP NE TEDD:
+
+```js
+// Példa
+const elem = <h1>Helló világ</h1>;
+ReactDOM.renderelés(elem, dokumentum.elemKiemeléseAzonosítóval('gyökér'));
+```
+
+## Külső linkek
+
+Ha egy külső link például egy az [MDN]-en vagy [Wikipédia]-n lévő cikkre mutat, és az adott cikk rendelkezik magyar fordítással, akkor azt a verziót linkeld.
+
+[MDN]: https://developer.mozilla.org/hu/
+[Wikipédia]: https://hu.wikipedia.org/wiki/Kezd%C5%91lap
+
+Példa:
+
+```md
+React is a [JavaScript library](https://en.wikipedia.org/wiki/JavaScript_library).
+```
+
+✅ OK:
+
+```md
+A React egy [JavaScript keretrendszer](https://hu.wikipedia.org/wiki/JavaScript_keretrendszer).
```

Comment:
```suggestion
A React egy [JavaScript könyvtár](https://hu.wikipedia.org/wiki/JavaScript_keretrendszer).
```

Reply:
Wikipedia information is wrong. He don't know the difference between library and framework  :tipping_hand_woman:

Reply:
Probably that is where my confusion came from! I blindly trusted the translation on the Wikipedia page. I tried to edit the source, we can update the link if it will be accepted: https://hu.wikipedia.org/wiki/JavaScript_könyvtár

Reply:
Great job ! Nice workaround :+1: